### ISSUE TITLE
[Mobile] If user clicks the "Submit Answer" button before selecting an answer or the "Answer Submitted" button after an answer has been selected, the app crashes (Android)

### ISSUE THREAD

**Issue by sincwu** [rightoneducation/righton-app#523]
![Screenshot_20230110-153952_RightOn](https://user-images.githubusercontent.com/8507656/211685173-51530c10-cc98-41c2-986c-a72d90c9ac6b.jpg)

**Comment by drewjhart** [comment]
Hi @josephinaim, can you double check that this is also relevant to iOS? If so, if you wouldn't mind working on this after [#511](https://github.com/rightoneducation/righton-app/issues/511), that'd be great.

**Comment by josephinaim** [comment]
This is what happens in iOS when I click the submit answer button before selecting an answer.
![Image](https://user-images.githubusercontent.com/79948102/211930834-4be2c665-dba0-40f1-abc7-249382a96c36.png)

**Comment by josephinaim** [comment]
Oh no, let me try again. Sorry about that!

**Comment by josephinaim** [comment]
It works on iOS now! The user can't click on the submit answer button before or after choosing an answer.

**Comment by drewjhart** [comment]
Beauty - thanks for the verification on that!
